plugins/UserId/Archiver.php

```diff
@@ -137,4 +150,15 @@ private function setVisitorIds(DataTable $dataTable)
         }
     }
 
+    private function getRankingQueryLimit()
+    {
+        $configGeneral = Config::getInstance()->General;
+        $configLimit = $configGeneral['archiving_ranking_query_row_limit'];
+        $limit = $configLimit == 0 ? 0 : max(
+            $configLimit,
+            $configGeneral['datatable_archiving_maximum_rows_standard']
```

Comment:
* let's introduce a different setting eg `datatable_archiving_maximum_rows_userid` 
* set this to a higher value like 5000 rows (i expect users will want to archive more rows in this report.)

Reply:
@mattab changed & build fixed. Had to add another order by to make the results deterministic.